### ISSUE TITLE
Create Top Bar Component

### ISSUE THREAD

**Issue by reeved** [Mafia-Hands/Mafia#23]
**What are the tasks that need to be done?**
- [x] Create Top Bar Component
- [x] Display Username
- [x] Display User Role
- [x] Add Timer Component
- [x] Create Settings Button

### In-Depth Feature Description
This will be the bar that is displayed above the primary game environment. It will always show the User Name, User Role, and a settings button. There will also be a Timer Component, but that will only be displayed in certain scenarios

**Dependencies:**
- Timer depends on #30 

### Images for Reference:
![image](https://user-images.githubusercontent.com/7234148/110555616-c46b0d80-81a1-11eb-859f-b4a4a778d909.png)

**Comment by reeved** [comment]
Closing this to create a new issue regarding adding the new Timer component from #30  into the TopBarGame component.
